script/build-app.sh

```diff
@@ -110,4 +111,10 @@ if [ "$1" == "--tarball" ]; then
   tar -zcvf $RELEASE_TARBALL $RELEASE_DIR
 fi
 
+if [ "$1" == "--zip" ]; then
+  pushd builds
+  "/cygdrive/c/Program Files/7-Zip/7z.exe" a $RELEASE_ZIP "$RELEASE/*"
```

Comment:
Since this is a cygwin specific path, could you make a note that this branch is windows-specific?